modules/minimongo-observable/observable-cursor.ts

```diff
@@ -79,13 +80,31 @@ export class ObservableCursor<T> extends Observable<T[]> {
     });
   }
 
+  addedAt(doc, at, before) {
+    this.data.splice(at, 0, doc);
+    this.handleChange();
+  }
+
+  changedAt(doc, old, at) {
+    this.data[at] = doc;
```

Comment:
You're right. I copied most of the implementation from a previous implementation of toObservable where my implementation of handleChange looks like this: 

```
	handleChange() {
		this.observer.next(this.data);
	};
```

Forgot to include that part.

The reasoning is very simple, since we recieve each modified entry and it's correct position we might as well store them to avoid the `fetch()`. Might cause some issues with invariance though so it might be an idea to switch to some other implentation in that case.